common/persistence/nosql/nosqlplugin/cassandra/gocql/session.go

```diff
@@ -48,7 +48,7 @@ type (
 	}
 )
 
-func newSession(
+func NewSession(
```

Comment:
why do we need to export this function? I think Client interface has a CreateSession method

Reply:
because the native client needs to call it to create a session. LMK if I missed anything.